samples/web-java-spring-petclinic/devfile.yml

```diff
@@ -0,0 +1,43 @@
+specVersion: 0.0.1
+name: spring-petclinic
+projects:
+  - name: spring-petclinic
+    source:
+      type: git
+      location: 'https://github.com/che-samples/web-java-spring-petclinic.git'
+tools:
+  - name: theia-ide
+    type: cheEditor
+    id: 'org.eclipse.che.editor.theia:1.0.0'
+  - name: exec-plugin
+    type: chePlugin
+    id: 'che-machine-exec-plugin:0.0.1'
+  - name: dev
+    type: dockerimage
+    image: registry.devshift.net/che/centos_jdk8
+    env:
+      - name: CHE_MACHINE_NAME
```

Comment:
Why is `CHE_MACHINE_NAME` env var needed here?
If it is used as an example of env var, I would choose another sample env var to avoid confusion.

Reply:
It was present in original sample https://github.com/che-samples/web-java-spring-petclinic/blob/master/.factory.json#L28  so i moved it here. Need to test if it works without it.

Reply:
Anyway for that example it was true, in case of Devfile machine name will be the equals to tool name `dev`